test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java

```diff
@@ -164,90 +216,49 @@ private static CommandGen<Spec> cqlOperations(Spec spec)
 
     private static Command<State<Spec>, Void, ?> validateAll(State<Spec> state)
     {
-        Spec spec = state.schemaSpec;
-        var schema = spec.harry.schema();
-        boolean writeThroughAccord = schema.isWriteTimeFromAccord();
+        Spec spec = state.testState;
         List<Command<State<Spec>, Void, ?>> reads = new ArrayList<>();
-        Model model = spec.harry.quiescentChecker();
-        for (Long pd : new TreeSet<>(spec.harry.pds()))
+
+        for (Integer pkIdx : spec.pkGen.generated())
         {
-            reads.add(new HarryCommand(s -> "Harry Validate pd=" + pd  + state.commandNamePostfix(), s -> model.validate(Query.selectAllColumns(schema, pd, false))));
-            // as of this writing Accord does not support ORDER BY
-            if (!writeThroughAccord)
-                reads.add(new HarryCommand(s -> "Harry Reverse Validate pd=" + pd + state.commandNamePostfix(), s -> model.validate(Query.selectAllColumns(schema, pd, true))));
+            long pd = spec.schema.valueGenerators.pkGen.descriptorAt(pkIdx);
+            reads.add(new HarryCommand(s -> String.format("Harry Validate pd=%d%s", pd, state.commandNamePostfix()), s -> spec.history.selectPartition(pkIdx)));
         }
         reads.add(new HarryCommand(s -> "Reset Harry Write State" + state.commandNamePostfix(), s -> ((HarryState) s).numInserts = 0));
         return Property.multistep(reads);
     }
 
-    private static class AccordSut extends InJvmSut
```

Comment:
how are you replacing this behavior?  I don't see you wrapping CQL with `BEGIN TRANSACTION` which would be a huge regression for this patch.  Please make sure `HarryOnAccord` always wraps every CQL in `BEGIN TRANSACTION`

Reply:
Wrapping is done for all transactions statements, even singluar ones in QueryBuildingVisitExecutor.

Reply:
where is this done, as this isn't the desired behavior when we finally add the `passthrough` test... and shouldn't happen when we do the Accord SAI tests

Reply:
```
public QueryBuildingVisitExecutor(SchemaSpec schema)
    {
        this.schema = schema;
        Object v = schema.options.get(SchemaSpec.Options.TRANSACTIONAL_MODE);
        if (v == null || v.equals("off"))
        {
            wrapSingleQueryFormat = "%s";
            wrapQueryFormat = "BEGIN UNLOGGED BATCH\n" +
                              "  %s\n" +
                              "APPLY BATCH;";
        }
        else
        {
            wrapQueryFormat = "BEGIN TRANSACTION\n" +
                              "  %s\n" +
                              "COMMIT TRANSACTION;";
            wrapSingleQueryFormat = wrapQueryFormat;
        }
    }
```

This is a bug... let me see how we opt-in to this behavior... opt-out would be an issue